zproject/prod_settings_template.py

```diff
@@ -105,6 +105,28 @@
 #PHYSICAL_ADDRESS = ''
 
 
+################
+# Email gateway integration.
+#
+# The email gateway integration supports sending messages into Zulip
+# by sending an email.
+# For details, see the documentation:
+#   https://zulip.readthedocs.io/en/latest/production/email-gateway.html
+EMAIL_GATEWAY_PATTERN = ""
+
+# If you are using polling, edit the IMAP settings below:
+#
+# The IMAP login; username here and password as email_gateway_password in
+# zulip-secrets.conf.
+EMAIL_GATEWAY_LOGIN = ""
+# The IMAP server & port to connect to
+EMAIL_GATEWAY_IMAP_SERVER = ""
+EMAIL_GATEWAY_IMAP_PORT = 993
+# The IMAP folder name to check for emails. All emails sent to EMAIL_GATEWAY_PATTERN above
+# must be delivered to this folder
+EMAIL_GATEWAY_IMAP_FOLDER = "INBOX"
```

Comment:
One downside of this change is that it moves this setting under the "Mandatory settings" heading.  We probably shouldn't move it unless our plan is to remove that heading (which might be a good idea at this point).

Reply:
Do we want to remove "Mandatory settings"?

Reply:
Can you start a thread on #backend in chat.zulip.org about this question?  I think it needs a bit of discussion.

Reply:
Created thread : https://chat.zulip.org/#narrow/stream/3-backend/topic/Remove.20.22Mandatory.20setting.22.3F